data/hai/hai missions.txt

```diff
@@ -1813,169 +1851,170 @@ mission "Nanachi Meeting"
+					defer
+			
+			label "know coalition"
+			`You've returned to <planet> to help the Hai to develop hull repair technology. Torsa's father asked if you knew of any aliens with hull repair technology, and you have met the Coalition, who have outfits capable of repairs hulls. Would you like to tell the elders about the Coalition?`
+			choice
+				`	(Yes.)`
+					goto start
+				`	(Not right now.)`
+					defer
+			
+			label "know remnant"
+			`You've returned to <planet> to help the Hai to develop hull repair technology. Torsa's father asked if you knew of any aliens with hull repair technology, and you have met the Remnant, who have ships that repair themselves. Would you like to tell the elders about the Remnant?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			
+			label start
+			`	You inform the elders that you have discovered a group that could help them with creating hull repair technology and they invite you to the council chamber. "Greetings, Captain <last>," one of the elders says to you upon entering the chamber. "What news do you have to bring us on your search for hull repair technology?"`
+			branch "tell both"
+				has "license: Remnant"
+				has "license: Coalition"
+			
+			branch "tell remnant"
+				has "license: Remnant"
+			
+			branch "tell coalition"
+				has "license: Coalition"
+			
+			label "tell both"
+			choice
+				`	"There is a group of three alien species far from here known as the Coalition that have outfits able to repair a ship in flight."`
+					goto coalition
+				`	"There is a group of humans far from here known as the Remnant who have ships that can repair themselves."`
+					goto remnant
+			
+			label "tell remnant"
+			choice
+				`	"There is a group of humans far from here known as the Remnant who have ships that can repair themselves."`
+					goto remnant
+			
+			label "tell coalition"
+			choice
+				`	"There is a group of three alien species far from here known as the Coalition that have outfits able to repair a ship in flight."`
+					goto coalition
+			
+			label remnant
+			apply
+				set "strider: remnant"
+			`	"A group of humans with hull repair technology?" one of the elders asks with a puzzled look. "I am surprised that we have not heard of these humans before. Are they well known among the Republic?"`
+			`	"No," you explain. "They are a reclusive group unknown to most of humanity. They likely wouldn't take kindly to being visited by aliens."`
+				goto end
+			
+			label coalition
+			apply
+				set "strider: coalition"
+			`	"A group of three aliens?" one of the elders asks with a puzzled look. "It is not often in the galaxy that such groups appear."`
+			
+			label end
+			`	"We will put together a delegation for you to transport to these people you mention. They will be waiting in the spaceport shortly," another elder says. "From there you can lead them how you see fit to make contact with this group. We trust that you will be capable in doing so."`
```

Comment:
```suggestion
			`	"We will put together a delegation for you to transport to these people you mention. They will be waiting in the spaceport shortly," another elder says. "You may lead them to make contact with this group how you see fit. We trust that you will be capable in doing so."`
```
This is optional. The phrasing of that second-to-last sentence is a little strange. Don't really know if this suggestion is better worse.